flake.nix

```diff
@@ -20,11 +20,6 @@
         }
       ) {
         enableSeparateDataOutput = false;
-      }).overrideAttrs (old: {
-        installPhase = old.installPhase + ''  
-          ln -s $out/bin/shellify $out/bin/nix-shellify
-        '';
-        buildInputs = old.buildInputs ++ [ pkgs.nix ];
```

Comment:
I removed adding `pkgs.nix` to `buildInputs` because it built fine without it.

Maybe your intention was to make `nix` available at runtime?

Reply:
Yes, the intention is to explicitly show that the tool has  a runtime dependency on nix. It calls `nix flakes registry list` when using new style commands to look up the registry URL from its name. Do you know a better way of showing this runtime dependency? I realise we all have `nix` in practice but I suppose it would be possible to use nix to deploy a system without nix. (Or if not it should be!)